### PR TITLE
Code base and test

### PR DESCRIPTION
**Pull request by trojkat**

### CODE REVIEW

**Comment by fizyk** [comment]
If you remember the source of solution, please add it to readme
Plus use case

**Comment by trojkat** [comment]
Updated.

File: AUTHORS.rst [hunk 1]
```diff
@@ -4,3 +4,5 @@ Authors
 This file contains the list of people involved in the development
 of pytest_repeater along its history.
 
+- Frank T (http://stackoverflow.com/users/1812727/frank-t)
```
Comment (fizyk) [inline AUTHORS.rst:7]: a link to thread would be nice to have as well within readme ;)

Comment (trojkat) [inline AUTHORS.rst:7]: Done.

**Comment by fizyk** [comment]
:+1:

**Pull request merged**